scipy/linalg/tests/test_procrustes.py

```diff
@@ -2,213 +2,209 @@
 
 import numpy as np
 import pytest
-from numpy.testing import assert_array_less, assert_allclose
+from numpy.testing import assert_allclose
 from pytest import raises as assert_raises
 
-from scipy.linalg import inv, eigh, norm, svd
 from scipy.linalg import orthogonal_procrustes
 from scipy.sparse._sputils import matrix
+from scipy._lib._array_api import make_xp_test_case
+from scipy._lib._array_api_no_0d import xp_assert_close
```

Comment:
Question: why is this needed? IIUC the `_no_0d` version is needed to for functions which emulate numpy reductions and return python floats instead of 0D arrays. Does something in linalg do this?